src/components/button/button.ts

```diff
@@ -37,15 +55,15 @@ export class MdButton {
   /** Whether a mousedown has occurred on this element in the last 100ms. */
   isMouseDown: boolean = false;
 
-  setClassList() {return `md-${this.color}`;}
```

Comment:
1. This results in an empty class attribute which is not valid XHTML
2. It overrides any other specified class in the component

`<button md-button class="myClass" color="warn">Submit</button>`
results in 
`<button md-button class="md-warn">Submit</button>`

It should be 
`<button md-button class="myClass md-warn">Submit</button>`